### DISCUSSION TITLE
SDKの規約と二次配布の範囲について

### DISCUSSION THREAD

**Discussion by Fuses-Garage** [pixiv/vroid-sdk-developers#135] [Q&A] [answered]
卒業制作にマルチプレイSDKを使っているのですが、卒業研究の成果物としてSDKを含むプロジェクトの提出を求められました。
これは規約違反や契約違反に当たってしまうのでしょうか？
心配です。

**Comment by nyamadan** [comment, marked as answer]
ビルドしたアプリを卒業制作として提出いただくことは問題ございません。
一方で、VRoid SDKを第三者に譲渡するのは[利用規約で禁止](https://policies.pixiv.net/#vroidsdk:~:text=%E7%AC%AC9%E6%9D%A1%20%E7%AC%AC%E4%B8%89%E8%80%85,%E3%81%AF%E3%81%AA%E3%82%8A%E3%81%BE%E3%81%9B%E3%82%93%E3%80%82)しております。

ですが、卒業制作ということでお力になりたく、以下の質問にお答えいただくことはできますか？

 - 提出いただくプロジェクトは（教員のみ・学内・学外など）どの範囲で共有となりますか？
 - シークレットやSDKのソースコードを除いた状態で提出いただくことは可能でしょうか？

**Reply by Fuses-Garage** [reply]
あれから教師を説得し、SDKを抜いた状態でのプロジェクトファイルの- 提出を認めてもらいました！答えていただきありがとうございます。
